File: weather-demo-app-apple/Weather/Domain/DTO/MainInfoDTO.swift
//
//  MainInfoDTO.swift
//  Weather
//
//  Created by Onur on 4/11/26.
//

struct MainInfoDTO: Decodable {
    let temp: Double
    let feelsLike: Double
    let humidity: Int

    enum CodingKeys: String, CodingKey {
        case temp
        case feelsLike = "feels_like"
        case humidity
    }
}

File: weather-demo-app-android/app/src/main/java/com/demo/weather/domain/dto/MainInfoDto.kt
package com.demo.weather.domain.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MainInfoDto(
    val temp: Double,
    @SerialName("feels_like")
    val feelsLike: Double,
    val humidity: Int
)